Updated OLAM cluster playbooks to work with OLAM 2.2 (#42)
oracle-samples/ansible-playbooks: playbooks/OLAM/cluster-plus-hop-node/README.md

1. Clone the repo:

    ```
    git clone https://github.com/oracle-samples/ansible-playbooks.git ol-playbooks
    cd ol-playbooks/playbooks/OLAM/cluster-plus-hop-node
    cp group_vars/all.yml.example group_vars/all.yml
    cp inventory/hosts.ini.example inventory/hosts.ini
    ```

> 1. Edit the group variables, minimum change passwords and ssh file:

Reviewer: ```suggestion
1. Edit the group variables, change the default passwords and replace the sample ssh key files:
```

    # Enter the non-hashed password for the non-opc user account.
    

>     "user_default_password": oracle

Reviewer: ```suggestion
    "user_default_password": CHANGE_ME
```

>     # Enter the password for postgress awx user

Reviewer: ```suggestion
    # Enter the password for PostgreSQL awx user
```

>     "awx_pguser_password": password

Reviewer: ```suggestion
    "awx_pguser_password": CHANGE_ME
```

>     # Enter the password for postgress awx user

Reviewer: ```suggestion
    # Enter the password for PostgreSQL awx user
```

>     "olam_admin_password": admin

Reviewer: ```suggestion
    "olam_admin_password": CHANGE_ME
```

>     # Enter the name of a local ssh keypair pub file located in the ~/.ssh directory. This key appends

Reviewer: ```suggestion
    # Enter the name of a local ssh public key file located in the ~/.ssh directory. This key appends
```
